types/audiosprite/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for audiosprite 0.6
+// Project: https://github.com/tonistiigi/audiosprite
+// Definitions by: Gyusun Yeom <https://github.com/Perlmint>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace audiosprite;
+export = audiosprite;
+
+declare function audiosprite(files: string[], option: audiosprite.Option, callback: (error: Error, obj: audiosprite.Result) => void): void;
+
+declare namespace audiosprite {
+    type ExportType = "jukebox" | "howler" | "createjs";
+    type LogLevel = "debug" | "info" | "notice" | "warning" | "error";
+    type VBR = -1 | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
+    type Channels = 1 | 2;
+    interface Option {
+        output?: string;
+        path?: string;
+        export?: string;
+        format?: ExportType;
+        log?: LogLevel;
+        autoplay?: string;
+        loop?: string[];
+        silence?: number;
+        gap?: number;
+        minlength?: string;
+        bitrate?: number;
+        vbr?: VBR;
```

Comment:
Missing `'vbr:vorbis'` key; it also looks like it can also accept `10`.

I am not sure if the VBR is necessarily an integer value for vorbis.